### PR TITLE
Labels implementation

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Comment by Thundermaker300** [comment]
Fix merge conflict :trollface:

**Comment by Thundermaker300** [comment]
Please also add label support to GOTOIF :heart:

**Comment by louis1706** [comment]
> Please also add label support to GOTOIF ❤️

it's should already work

**Comment by Thundermaker300** [comment]
Fix merge conflicts

File: ScriptedEvents/API/Helpers/ScriptHelper.cs [hunk 1]
```diff
@@ -70,6 +73,15 @@ public static Script ReadScript(string scriptName)
                 string[] actionParts = action.Split(' ');
                 string keyword = actionParts[0].RemoveWhitespace();
 
+                // Labels
+                if (keyword.EndsWith(":"))
+                {
+                    script.Labels.Add(action.Remove(keyword.Length - 1, 1), currentline);
+                    script.Actions.Add(new NullAction());
```
Comment (Thundermaker300) [inline ScriptedEvents/API/Helpers/ScriptHelper.cs]:
```suggestion
                    script.Actions.Add(new NullAction($"{action.Remove(keyword.Length - 1, 1)} LABEL"));

```
Or make a variable in the if statement that's used in both places

**Pull request merged**